packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -331,6 +331,9 @@ class Requests {
     public cannotShareRequestWithYourself() {
         return new CoreError("error.consumption.requests.cannotShareRequestWithYourself", "You cannot share a Request with yourself.");
     }
+    public cannotCreateRequestWithExpirationDateInPast() {
```

Comment:
newline

Reply:
woops already did that but forgot to push